.gitlab-ci.yml

```diff
@@ -166,7 +166,8 @@ variables:
   DATADOG_AGENT_NIKOS_BUILDIMAGES: v20878799-a2f77ae
   DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
   DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v20878799-a2f77ae
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 40c538510ddf
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: "-dev"
```

Comment:
Should be removed before merging on `main`

Reply:
> Will update the test-infra-definitions version once https://github.com/DataDog/test-infra-definitions/pull/388 is merged.

I did mention that in the PR description

Reply:
Updated